### PR TITLE
Feat/wrap unwrap

### PR DESCRIPTION
**Pull request by glensk**
This should be solely the wrap/unwrap changes.

### CODE REVIEW

File: tools/py/mux-positions.py [hunk 1]
```diff
@@ -4,21 +4,61 @@
...
+    cell = frame['cell'].h
+    cell_inv = np.linalg.inv(cell)
+    s = cell_dot_pos(cell_inv,pos)
+    sc = s - np.round(s)
+    frame['atoms'].q = cell_dot_pos(cell,sc).flatten()
+    return frame
+
+
+def unwrap_positions(frame,framelast):
+    frame_wr = wrap_positions(frame)
```
Comment (ceriottm) [inline tools/py/mux-positions.py]: Why do you wrap before unwrapping?

File: tools/py/mux-positions.py [hunk 2]
```diff
@@ -53,8 +94,16 @@ def main(fns_in, fn_out, begin, end, stride):
...
+            for trj in trjs_in:                
                 frame = trj.next()
+
+                if wrap:
+                    frame = wrap_positions(frame)
+                if unwrap:
+                    frame = unwrap_positions(frame,frame_last)
+
+
+                frame_last = frame
```
Comment (ceriottm) [inline tools/py/mux-positions.py]: There can be multiple trajectories, so you should store a frame_last for each trajectory. 
Something like holding a list sized as trjs_in, and traversing in together with trjs_in.

File: tools/py/mux-positions.py (hunk 1 above)
Comment (glensk) [reply tools/py/mux-positions.py]: did remove the wrap before unwrapping. Also, put wrap/unwrap to the bottom. Created separate functions for wrap unwrap due to your suggestion above: 
if wrap:
    wrap positions
if unwrap:
    unwraps positions

File: tools/py/mux-positions.py (hunk 2 above)
Comment (glensk) [reply tools/py/mux-positions.py]: done.

**Pull request closed without merging**